src/assets/index.js

```diff
@@ -2,11 +2,17 @@ import react from "./react.svg";
 import vite from "./vite.svg";
 import menuIcon from "./Menu.svg";
 import close from "./close.svg";
-import github from "./github.png";
 import search from "./Search.svg";
 import downArrow from "./down-arrow.svg";
 import colorLogo from "./brand/yumyumyes-logo-m.png";
 import creamLogo from "./brand/yumyumyes-logo-lightcream.svg";
+import bannaLabel from "./HomePage/bananaLabel.png";
```

Comment:
it could be this spelling here. 
```suggestion
import bananaLabel from "./HomePage/bananaLabel.png";
```